### PR TITLE
NCL4839 - Simplify code

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,4 @@ ObjectStore
 *.hprof
 /.nb-gradle/
 
+*.plantuml
```
Comment (geoand) [inline .gitignore]: Just out of curiosity, what is this for?

File: analyzer/build.gradle.kts [hunk 2]
```diff
@@ -1,9 +1,5 @@
 group = "org.jboss.gm.analyzer"
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
```
Comment (geoand) [inline analyzer/build.gradle.kts]: Is this the default?

Comment (geoand) [reply analyzer/build.gradle.kts]: Saw the later configuration so ignore this :)

File: .gitignore (hunk 1 above)
Comment (rnc) [reply .gitignore]: Ooops, didn't mean to commit that. Admittedly its harmless - its for ignoring the diagrams generated by https://plugins.jetbrains.com/plugin/10387-sketch-it-/

File: analyzer/src/test/java/org/jboss/gm/analyzer/alignment/DependencyOverrideCustomizerTest.java [hunk 3]
```diff
@@ -24,12 +22,21 @@ public void ensureOverrideOfDependenciesWorks() {
...
+        Response originalResp = new Response(
+                new HashMap<ProjectVersionRef, String>() {
+                    {
+                        put(hibernateGav, hibernateGav.getVersionString() + DEFAULT_SUFFIX);
+                        put(undertowGav, undertowGav.getVersionString() + DEFAULT_SUFFIX);
+                    }
+                });
+        originalResp.setNewProjectVersion(newProjectVersion);
+        /*
         when(originalResp.getAlignedVersionOfGav(any(ProjectVersionRef.class))).thenAnswer((Answer<String>) invocation -> {
```
Comment (geoand) [inline analyzer/src/test/java/org/jboss/gm/analyzer/alignment/DependencyOverrideCustomizerTest.java]: This part is supposed to be removed completely I assume, correct?

File: analyzer/src/test/java/org/jboss/gm/analyzer/alignment/UpdateProjectVersionCustomizerTest.java [hunk 4]
```diff
@@ -41,14 +38,19 @@
...
     @Test
-    public void ensureProjectVersionIsUpdatedWhenOriginalResponseHasNoProjectVersion() throws IOException {
-        final AlignmentService.Response originalResp = mock(AlignmentService.Response.class);
+    public void ensureProjectVersionIsUpdatedWhenOriginalResponseHasNoProjectVersion()
+            throws IOException, ManipulationException {
+        final Response originalResp = new Response();
+
+        /*
         // just return whatever was passed
         when(originalResp.getAlignedVersionOfGav(any(ProjectVersionRef.class))).thenAnswer((Answer<String>) invocation -> {
```
Comment (geoand) [inline analyzer/src/test/java/org/jboss/gm/analyzer/alignment/UpdateProjectVersionCustomizerTest.java]: Same here

File: analyzer/src/test/java/org/jboss/gm/analyzer/alignment/UpdateProjectVersionCustomizerTest.java [hunk 5]
```diff
@@ -60,21 +62,25 @@ public void ensureProjectVersionIsUpdatedWhenOriginalResponseHasNoProjectVersion
...
 
     @Test
-    public void ensureProjectVersionIsUpdatedWhenOriginalResponseHasNoProjectVersion2() throws IOException {
-        final AlignmentService.Response originalResp = mock(AlignmentService.Response.class);
+    public void ensureProjectVersionIsUpdatedWhenOriginalResponseHasNoProjectVersion2()
+            throws IOException, ManipulationException {
+        final Response originalResp = new Response();
+        /*
         // just return whatever was passed
         when(originalResp.getAlignedVersionOfGav(any(ProjectVersionRef.class))).thenAnswer((Answer<String>) invocation -> {
```
Comment (geoand) [inline analyzer/src/test/java/org/jboss/gm/analyzer/alignment/UpdateProjectVersionCustomizerTest.java]: And here

**Comment by geoand** [comment]
Looks great! I just added some minor comments

**Pull request merged**